### ISSUE TITLE
PaymentsMeans node Does not support multiple PaymentMeans. 

### ISSUE THREAD

**Issue by saqijaan** [josemmo/einvoicing#62]
PaymentMeans on peppol BIS is 0..n which means we can add multiple payment means and [Payment](cac:PayeeFinancialAccount) only support 0..1 nodes. 

Currenlty library suports multiple cac:PayeeFinancialAccount nodes but only one PaymentMeans code which is invalid as we want to add multiple PaymentMeans.

**Comment by yurikuzn** [comment]
I was curious about this problem too.

Maybe it would be reasonable to add `addPayment` method and keep `getPayment` and `setPayment` for backward compatibility.

**Comment by josemmo** [comment]
ok, since everyone keeps asking for this feature, I'll add it.

**Comment by josemmo** [comment]
I just pushed an update to develop implementing this feature. Could you give it a try and confirm that it works as intented?

```
composer require josemmo/einvoicing:dev-develop
```

**Comment by yurikuzn** [comment]
I tried the update. Tried using the *addPayment* method. Looks fine for me. But I have no enough expertise in E-Invoicing to say for sure. Thanks for your work.

**Comment by saqijaan** [comment]
@josemmo I'll try it today or tomorrow.

**Comment by yurikuzn** [comment]
The PR looks fine to me. Can we expect it released? Thanks.

**Comment by josemmo** [comment]
You're right @yurikuzn, I was waiting on the response from the OP and completely forgot.
Will merge it to the main branch ASAP after performing some maintenance tasks on the library.

**Comment by josemmo** [comment]
Done! https://packagist.org/packages/josemmo/einvoicing#v0.3.0
